packages/bot-web-ui/src/components/bot-notification/bot-notification.scss

```diff
@@ -1,15 +1,24 @@
-.bot-snackbar {
+.bot-notification {
     position: fixed;
     z-index: var(--zindex-snackbar);
-    right: 38rem;
-    top: 12rem;
+    left: 3.5rem;
+    bottom: 7rem;
 
     .dc-toast {
         width: 100%;
         &__message {
-            background: var(--text-prominent);
+            background: var(--icon-black-plus);
             color: var(--general-main-1);
-            padding: 1rem 1.6rem;
+            height: 4.8rem;
+            text-align: left;
```

Comment:
@farabi-deriv can we go with `text-align: start` here? Same reason as above because of RTL support.

Reply:
sure ✅